components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -0,0 +1,110 @@
+
+![Chef Automate](/images/automate/chef-automate-on-chef-infra-page.png)
+
+Click on _Infrastructure_ located at the top navigation tab of the page. Then select _Chef Servers_ located at the left panel of the page. The page looks like as shown below:
+
+![Chef Infra Server](/images/automate/chef-server-page.png)
+
+The above image is the _Chef Server_ page which contains all the Chef Infra Servers connected to the Chef Automate. The Chef Infra Servers listed are the ones that are already configured and added to the Chef Automate. To create a new Infra Server click [here](https://docs.chef.io/automate/infra_server/).
+
+To add the created Chef Server to the Infrastructure, click on _Add Chef Server_ button. Clicking on the Add Chef Server button opens a popup menu as shown below:
+
+![Add Chef Server Popup Menu](/images/automate/add-chef-server-popup-menu.png)
+
+Mention the correct Name, FQDN, and IP Address of the system. Click _Add Chef Server_. The desired server gets added to the list of  Chef Infra Server.
+
+**Note:** If the mentioned FQDN and IP Address is incorrect, the box displays an error as shown below:
+
+![Add Chef Server Popup Menu](/images/automate/add-chef-server-popup-menu-with-error.png)
+
+Click on the listed Servers to view the details of the specific _Chef Server_.
+
+## Connect to the Chef Organization
+
+Selecting a specific added Chef Server opens the list of organizations in the Server. Refer to the image below:
+
+![Chef Infra Server Organization](/images/automate/chef-server-organization.png)
+
+The Organization in the Chef server listed are the ones which are already created in that specific server. To create a new Organization in the Server click [here](https://docs.chef.io/automate/infra_server/#set-up-the-chef-infra-server).
+
+To add the previously created organization to the Infrastructure, click **Add Chef Organization** button. Clicking on the button opens a popup menu as shown below:
+
+![Add Chef Organization Popup Menu](/images/automate/add-chef-organization-popup-menu.png)
+
+Add the Name, Projects, Admin User, and Admin Key. Always use the exact details that were provided at the time of setting up _Knife_. Copy the contents of the `~/.chef/USER.pem` file and paste it into the _Admin Key_ field. Once done click **Add Chef Organization**. The desired organization will be added to the Chef Server.
+
+Click on any Chef Organization to view the _Objects_.
+
+## Access Chef Infra Server Objects
+
+The Chef Infra Server Objects can be managed from the Chef Automate. In Automate Infra views, following objects can be managed:
+
+- Cookbooks
+- Roles
+- Environments
+- Data Bags
+- Clients
+
+Refer to the image below:
+
+![Chef Infra Server Objects](/images/automate/chef-infra-server-objects.png)
+
+### Cookbooks
+
+A [cookbook](https://docs.chef.io/cookbooks/) is the fundamental unit of configuration and policy distribution. Include recipes and other files, components, or directories.
```

Comment:
The links should use relative URLs. Ideally with [relref](https://gohugo.io/content-management/cross-references/) so we'll know if a link breaks if page changes or gets deleted. So `[cookbook](https://docs.chef.io/cookbooks/)` should be formatted as `[cookbook]({{< relref "cookbooks" >}})`.